server/routes/resources.js

```diff
@@ -2,16 +2,61 @@ const router = require('express').Router();
 
 const Resources = require('../db/models/resources');
 
-router.get('/', (req, res) => {
-  Resources.find({})
-    .then(result => res.json(result))
-    .catch(() => console.log('Unable to fetch resources'));
+router.post('/', async (req, res) => {
+  try {
+    const { resourceName, comments } = req.body;
```

Comment:
This would be for a new resource so I dont think you need to include comments.
Comments would be added after this stage via the /resources/:id/comments route.
Just have it default to an empty array in the schema.